src/entities/applet/model/services/RefreshService.test.js

```diff
@@ -0,0 +1,133 @@
+import { QueryClient } from '@tanstack/react-query';
+import RefreshService from './RefreshService';
+import mock from './mockData';
+
+const MOCK_ACTIVITY_KEY = ['activities', { activityId: 'activity_test_id' }];
+const MOCK_APPLET_KEY = ['applets', { appletId: 'applet_test_id' }];
+const MOCK_APPLETS_KEY = ['applets'];
+const MOCK_EVENTS_KEY = ['events', { appletId: 'applet_test_id' }];
+
+jest.mock('@app/shared/api', () => ({
+  ActivityService: {
+    getById: jest.fn(() => mock.activityData),
+  },
+  AppletsService: {
+    getAppletDetails: jest.fn(() => mock.appletDetails),
+    getApplets: jest.fn(() => mock.applets),
+  },
+  EventsService: {
+    getEvents: jest.fn(() => mock.eventResponse),
+  },
+}));
+
+jest.mock('@app/shared/lib', () => ({
+  getActivityDetailsKey: jest.fn(() => MOCK_ACTIVITY_KEY),
+  collectActivityDetailsImageUrls: jest.fn(() => []),
+  collectAppletDetailsImageUrls: jest.fn(() => []),
+  getAppletDetailsKey: jest.fn(() => MOCK_APPLET_KEY),
+  getEventsKey: jest.fn(() => MOCK_EVENTS_KEY),
+  getAppletsKey: jest.fn(() => MOCK_APPLETS_KEY),
+  isAppOnline: jest.fn(() => Promise.resolve(false)),
+  onNetworkUnavailable: jest.fn(),
+}));
```

Comment:
I get why `isAppOnline` and `onNetworkUnavailable` are mocked but the rest of them don't have to be mocked as they return simple JS readable data.